### PR TITLE
[TW-1117] Collection backup integrations autosave

### PR DESCRIPTION
**Pull request by MatthewPinsker-postman**
* Postman regularly checks backed up collections for changes. If there are changes, then Postman backs up the changes.
* Removed the mention about this behavior from the Autosave option in Settings.

Note: It's my understanding that teams with team or collection activity integrations will receive notifications when collections are autosaved. Chose to not mention this in the documentation because the team/collection activity integrations don't explain every scenario that triggers a notification.

---

[Beta site](https://tw-1117-collection-backup-integration-autosave.learning.postman-beta.com/docs/introduction/overview/)

### CODE REVIEW

**Review by mark-anthony-postman — APPROVED** [review]
LGTM!

File: src/pages/docs/integrations/available-integrations/apimatic.md [hunk 1]
```diff
@@ -51,6 +51,6 @@ Once the integration is created, select it to view details about the integration
 
 <img alt="APIMatic integration run log" src="https://assets.postman.com/postman-docs/apimatic-run-log-v9.jpg"/>
 
-Changes made to your collection are picked up by the converter in its next run. If no updates are made to your collection, no updates are pushed to GitHub.
+Postman often checks your collection for changes. If Postman identifies changes to your collection, the changes automatically commit to your repository in JSON format.
```
Comment (saialekhya-001) [inline src/pages/docs/integrations/available-integrations/apimatic.md:54]: Can we rephrase this in such a way which communicates "backup is made periodically not immediately after collection changes are made"?

Comment (MatthewPinsker-postman) [reply src/pages/docs/integrations/available-integrations/apimatic.md]: Yes! I updated all occurrences of this content. Thanks!

**Review by mark-anthony-postman — APPROVED** [review]
LGTM!

**Review by shubhbhargav — APPROVED** [review]

**Review by saialekhya-001 — APPROVED** [review]

**Pull request merged**